.circleci/config.yml

```diff
@@ -20,14 +20,30 @@ jobs:
           command: |
             sudo apt-get update
             sudo apt-get install ghostscript libpng-dev imagemagick ffmpeg libreoffice dcraw
-      - setup_remote_docker
+      - restore_cache:
+          name: Restore Kakadu Cache
+          keys:
+            - kakadu-cache-v1
+      - run:
+          name: Download Kakadu
+          command: |
+            if [ ! -d "kakadu" ]; then
+              mkdir -p ~/downloads
+              wget http://kakadusoftware.com/wp-content/uploads/KDU805_Demo_Apps_for_Linux-x86-64_200602.zip -O ~/downloads/kakadu.zip
+              unzip ~/downloads/kakadu.zip
+              mv KDU805_Demo_Apps_for_Linux-x86-64_200602 kakadu
+            fi
+      - save_cache:
+          name: Save Kakadu Cache
+          key: kakadu-cache-v1
+          paths:
+            - ~/downloads/kakadu.zip
```

Comment:
It looks like the run command is checking for the existence of the kakadu directory but only the zip file is being cached so I think the zip will be downloaded every time.